### ISSUE TITLE
Function provisioned concurrency unable to reach minimum

### ISSUE THREAD

**Issue by L0key** [neiman-marcus/serverless-provisioned-concurrency-autoscaling#37]
With the following configuration:
```
plugins:
  - serverless-provisioned-concurrency-autoscaling
 ...
custom:
  provisionedConcurrency:
    ci: false
    uat: false
    prod: true
...
 functions:
  my_func:
    handler: handlers/my-handler
    provisionedConcurrency: 1
    concurrencyAutoscaling: ${self:custom.provisionedConcurrency.${opt:stage}} 
 ```

I was expecting to see the Lambda function's provisioned concurrency go to 1 whenever activity went to zero. It turns out the actual minimum is 2. I did check that the CloudWatch alarm for down-scaling the provisioned concurrency is in alarm and is enabled:
```
ProvisionedConcurrencyUtilization < 0.675 for 15 datapoints within 15 minutes
```

**Comment by L0key** [comment]
Follow-up question: Would it be possible to take minimums to zero?

**Comment by claydanford** [comment]
Hi @L0key, thanks for the issue. Do you mind elaborating on the minimum being 2?

It is not possible to go to zero, without scheduling, which is being worked on in another PR.

**Comment by L0key** [comment]
Sure thing @claydanford. After implementing the plugin, we've noticed that the provisioned concurrency on any given function never attains the set minimum of 1. The actual provisioned concurrency minimum is 2. I've attached a bit of evidence from one of the functions which is using the previously mentioned configuration:
<img width="1228" alt="Screen Shot 2022-03-08 at 8 26 50 AM" src="https://user-images.githubusercontent.com/2266671/157257716-83fd6877-4ca4-4ace-8bb6-499ad2bae1a4.png">
<img width="1342" alt="Screen Shot 2022-03-08 at 8 25 54 AM" src="https://user-images.githubusercontent.com/2266671/157257723-acc1dbf8-d8b7-4801-ab02-b93c52722eb5.png">

**Comment by claydanford** [comment]
Hi @L0key I was able to recreate the issue. I found that the scaleInCooldown is different than the AWS default. I published v1.7.0 in response. Now I see it scaling up and down. Do you mind updating and testing again?

**Comment by L0key** [comment]
Definitely! Thanks for being so responsive on this.

**Comment by cookieMr** [comment]
> published v1.7.0 in response

do you mean 1.9.0?

**Comment by nmrad-91** [comment]
> 

@claydanford created v1.7.0 to resolve this issue. v1.7.0 was superceded by v1.8.0 which introduced breaking changes and was reverted by creating v1.9.0. In essence v1.7.0 and v1.9.0 are the same. This issue should have been closed when v1.7.0 was created, but I was a slacker that day and did not close it then.

**Comment by L0key** [comment]
I was able to confirm that this now works.
